tests/schema_test.go

```diff
@@ -32,9 +32,10 @@ type Log struct {
 }
 
 type Stat struct {
-	ID  int64         `db:"id,omitempty"`
-	Key string        `db:"key"`
-	Num dbtype.BigInt `db:"big_num"` // using NUMERIC(78,0) postgres datatype
+	ID     int64          `db:"id,omitempty"`
+	Key    string         `db:"key"`
+	Num    dbtype.BigInt  `db:"big_num"` // using NUMERIC(78,0) postgres datatype
+	Rating *dbtype.BigInt `db:"rating"`  // using NUMERIC(78,0) postgres datatype
```

Comment:
this is the problematic runtime type, of *dbtype.BigInt .. resulting in the error on rows.Scan(..) of:

```
=== RUN   TestRowsWithBigInt
(*dbtype.BigInt)(nil)
    pgkit_test.go:379: 
        	Error Trace:	pgkit_test.go:379
        	Error:      	Received unexpected error:
        	           	pgkit: scany: scan row into struct fields: can't scan into dest[3]: unable to assign to *dbtype.BigInt
        	Test:       	TestRowsWithBigInt
    pgkit_test.go:382: 
        	Error Trace:	pgkit_test.go:382
        	Error:      	Should be true
        	Test:       	TestRowsWithBigInt
```